webpack/webpack.config.js

```diff
@@ -65,6 +65,12 @@ module.exports = {
   ],
   module: {
     rules: [
+      {
+        type: "javascript/auto",
```

Comment:
Would that transpile all the modules? Do we need this or it's more like a nice to have?

Reply:
Not a long term solution, but a workaround owing the to the fact that webpack 4 prefers mjs files to js files and react-apollo fails to account for this. https://github.com/apollographql/react-apollo/issues/1737